Makefile

```diff
@@ -2,7 +2,10 @@ PREFIX ?= /usr/local
 SYSCONFDIR ?= $(PREFIX)/etc
 
 
-all:
+all: check
+
+check:
+	find etc-conf -name '*.pem' -print | xargs -t -I{} openssl x509 -in {} -noout -checkend 0
```

Comment:
Shouldn't we rather call openssl with higher value of `-checkend`? I would say that new certificate should be valid at least for 5 years (It is 157680000 seconds).

Reply:
I don't think we should check for time validity in this case. We might be very well use certificates with still less than 5 years of validity, and still ship them. The logic whether a certificate is valid in the next future is sadly outside this own repository/source, it is more in the hands of whoever creates those certificates. Hence, checking for a date here would lead to potential test suite failures in the future in case the certificates are still acceptable.